### PR TITLE
nitrogen: deprecate

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- Deprecate nitrogen
- Resolves https://github.com/getsolus/packages/issues/2625

**Test Plan**
- Ran script, checked files were deleted.

**Checklist**

- [X] Package was built and tested against unstable

### CODE REVIEW

File: repo_data/distribution.xml.in [hunk 1]
```diff
@@ -3568,5 +3568,9 @@
 		<Package>libgpod-devel</Package>
 		<Package>libgpod-dbginfo</Package>
 
+		<!-- Dead upstream -->
+		<Package>nitrogen</Package>
+    <Package>nitrogen-dbginfo</Package>
```
Comment (davidjharder) [inline repo_data/distribution.xml.in:3573]: The whitespace here is funky, should be tabs, not spaces.

**Review by davidjharder — CHANGES_REQUESTED** [review]
Whitespace issue.
Neeeds a rebase as well now.

**Comment by davidjharder** [comment]
Sorry, this needs a rebase again

**Comment by androidnisse** [comment]
> Sorry, this needs a rebase again

Done.

**Review by davidjharder — APPROVED** [review]
Thanks!

**Pull request merged**